docs/nx-cloud/reference/nx-cloud-cli.md

```diff
@@ -1,5 +1,30 @@
 # `nx-cloud` CLI
 
+## npx nx-cloud login
+
+To provision a local personal access token to access Nx Cloud features run `npx nx-cloud login`. This will open your browser at the Nx Cloud application and after signing in will generate a personal access token and save it in a configuration file locally called `nxcloud.ini`. The format of this file will look like this:
+
+```ini
+[https://cloud\.nx\.app]
+personalAccessToken=SOME_ACCESS_TOKEN
+```
+
+If you have access to multiple instances of the Nx Cloud application (e.g. self-hosted Enterprise and our managed instance), each instance will be saved to this file under its URL.
+
+Our managed instance at https://cloud.nx.app is the default. To provision a personal access token from an alternative instance of Nx Cloud pass the URL as a positional parameter, e.g.
+
+```bash
+nx-cloud login https://nx-cloud.my-domain.app
+```
+
+## npx nx-cloud configure
+
+To provision more than one personal access token for multiple contexts (e.g. home and work machines) you can use the personal access tokens page under your Nx Cloud profile. To save a personal access token to your local `nxcloud.ini` file without needing to edit the file yourself call `nx-cloud configure` like this:
+
+```bash
+npx nx-cloud configure --personalAccessToken=SOME_ACCESS_TOKEN
+```
+
```

Comment:
By default, `nx-cloud configure` will save a personal access token under `https://cloud.nx.app`. Use `--nx-cloud-url` to  save a personal access token for another instance. 

```bash
npx nx-cloud configure --personalAccessToken=SOME_ACCESS_TOKEN --nx-cloud-url=https://nx-cloud.my-domain.app
```